vitest.config.mts

```diff
@@ -4,6 +4,7 @@ import packageJson from './package.json';
 export default defineConfig({
   test: {
     name: packageJson.name,
+    environment: 'happy-dom',
```

Comment:
Set up `happy-dom` with vitest's environment for accurate testing. 

We can do that and have a more accurate test for `isBrowser` as well.

Reply:
We could also use `jsdom`, but we already had `happy-dom` installed, and `jsdom` has issues with the `Blob.size` test.